src/components/DailyRoutine/DailyRoutine.tsx

```diff
@@ -30,7 +30,15 @@ const DailyRoutine: React.FC<Props> = ({ routine, onDeleteRoutine, onClickOpenEd
   return (
     <Styled.RoutineItem>
       <input type='checkbox' onChange={handleChangeCheckbox} defaultChecked={completed} />
-      <Styled.RoutineInfo isChecked={completed}>{startTime} ☀️ {title} {time}min</Styled.RoutineInfo>
+      {(startTime >= '05:00' && startTime <= '19:00') ?
+        <Styled.RoutineInfo isChecked={completed}>
+          {startTime} 🌞 {title} {time}min
+        </Styled.RoutineInfo>
+        :
+        <Styled.RoutineInfo isChecked={completed}>
+          {startTime} 🌚 {title} {time}min
+        </Styled.RoutineInfo>
+      }
```

Comment:
```suggestion
      {
        <Styled.RoutineInfo isChecked={completed}>
          {startTime} {(startTime >= '05:00' && startTime <= '19:00') ? 🌞 : 🌚} {title} {time}min
        </Styled.RoutineInfo>
      }
```
실제 변하는 부분에 예외처리를 해주는게 좋을 것 같네요.

Reply:
실제 변하는 부분이 뭔가요,?

Reply:
저 예외처리를 통해 실제로 화면이 변하는 부분을 의미했습니다~ 여기선 이모티지가 해당하겠네요.